client/src/app/ui/modules/grid/grid/grid.component.html

```diff
@@ -7,7 +7,7 @@ <h3>{{ title }}</h3>
 
 <div class="os-grid" [ngClass]="{ 'no-space': noSpace }">
     @for (tile of tiles; track tile) {
-        <div tabindex="-1" [ngClass]="tile.classes" (click)="tile.onClick($event)">
+        <div tabindex="0" [ngClass]="tile.classes" (click)="tile.onClick($event)" (keyup.enter)="tile.onClick($event)">
```

Comment:
os-gris:
motion-list -> works
meeting-settings -> now has an additional empty focusable item
projector-detail -> now has an additional empty focusable item

Please remove the additional item which is now focusable 
For the projector detail it is just ugly
For the meting settings this is confusing and will be frustrating if used. The visuals between the item which helps you and the one which does nothing is almost identical